cmd/skopeo/inspect.go

```diff
@@ -85,9 +96,25 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		return err
 	}
 
-	src, err := parseImageSource(ctx, opts.image, imageName)
-	if err != nil {
-		return fmt.Errorf("Error parsing image name %q: %v", imageName, err)
+	if _, err := alltransports.ParseImageName(imageName); err != nil {
```

Comment:
This is to avoid retries, because `isRetryable` returns `true` on unknown errors, isn’t it?

I’d imagine `isRetryable` should not match such these errors. (Otherwise we would pretty soon have to add a a  `neverRetry struct { err error }` wrapper and start adding it all over the place.)

Reply:
> This is to avoid retries, because `isRetryable` returns `true` on unknown errors, isn’t it?
> 

Yes. This also because I think it's `imageName` argument validation